libs/libnvme/Makefile

```
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 Luca Barbato
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=libnvme
+PKG_VERSION:=1.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=libnvme-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/linux-nvme/$(PKG_NAME)/archive/refs/tags/v$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/libnvme-$(PKG_VERSION)
+PKG_HASH:=f73ba1edde059b2d5e7c1048ad4f895e6047bff241c94b34a7aff5894779d086
+
+PKG_MAINTAINER:=Luca Barbato <lu_zero@luminem.org>
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/meson.mk
+
+define Package/libnvme
+  TITLE:=libnvme
+  URL:=https://github.com/linux-nvme/libnvme
+  SECTION:=utils
+  CATEGORY:=Libraries
+  DEPENDS:=+libjson-c
+  SUBMENU:=Filesystem
+  LICENSE:=GPL-2.0-or-later
+  LICENSE_FILES:=LICENSE
```

```suggestion
  SECTION:=utils
  CATEGORY:=Libraries
  SUBMENU:=Filesystem
  TITLE:=C Library for NVM Express on Linux
  URL:=https://github.com/linux-nvme/libnvme
  DEPENDS:=+libjson-c
```

sort, and convert LICENSE* to PKG_LICENSE*